packages/langium/test/utils/cst-utils.test.ts

```diff
@@ -0,0 +1,28 @@
+/******************************************************************************
+ * Copyright 2022 TypeFox GmbH
+ * This program and the accompanying materials are made available under the
+ * terms of the MIT License, which is available in the project root.
+ ******************************************************************************/
+import { createLangiumGrammarServices, EmptyFileSystem, Grammar } from '../../src';
+import { parseHelper } from '../../src/test';
+import * as c from '../../src/utils/cst-util';
```

Comment:
Minor: The name is a bit non-descriptive.
```suggestion
import * as cstUtil from '../../src/utils/cst-util';
```

Reply:
fixed